cmd/starbridge/integrations/ethereum.go

```diff
@@ -47,41 +48,60 @@ func Ethereum2Transaction(conn *ethclient.Client, txReceipt *types.Receipt, tx *
 	if !IsMyContractAddress(tx.To().Hex()) {
 		return nil, fmt.Errorf("unsupported receiver address '%s'", tx.To().Hex())
 	}
-
 	fromAddress, e := getFromAddress(tx)
 	if e != nil {
 		return nil, fmt.Errorf("unable to get From address: %s", e)
```

Comment:
💡 If you use `%w` then the fmt.Errorf func will wrap the error rather than stringify it.
```suggestion
		return nil, fmt.Errorf("unable to get From address: %w", e)
```